docs/server/README.md

```diff
@@ -1,14 +1,15 @@
 # Discodo Server Documentation
 
-## Urls
+## BaseUrls
+`--port PORT, -P PORT the port of the server (default: 8000)`
 
 Restful
 ```
- http://localhost
+ http://<host>:<port>
```

Comment:
missing root /

Reply:
``http://<host>:<port>/``

Reply:
http://<host>:<port>/

right? @sannoob

Reply:
BaseURL should not end with /.
Because endpoints are starting with /.

LGTM for me.

Reply:
hmmm

Reply:
LGTM